src/components/common/Extensions/UserNavigation/partials/UserSubmenu/UserSubmenu.tsx

```diff
@@ -11,7 +11,10 @@ const UserSubmenu: FC<UserSubmenuProps> = ({ submenuId }) => (
   <ul className="mt-2">
     {userSubmenuItems[submenuId].map((item) => (
       <li key={item.id} className="mb-4 last:mb-0">
-        <NavLink to={item.url} className="flex items-center">
+        <NavLink
+          to={item.url}
+          className="flex items-center transition-all duration-normal hover:text-blue-400"
```

Comment:
You don't these styles here `transition-all duration-normal hover:text-blue-400` because there are used in NavLink directly

Reply:
Removed